### PR TITLE
do not inject properties directly in module

### LINKED ISSUE #2687: Do not inject properties directly in module
As per the @drcrallen [comment](https://github.com/druid-io/druid-api/pull/76#issuecomment-198540198)
if you have a module like io.druid.server.initialization.jetty.ChatHandlerServerModule where the properties are injected, and the Module list in there takes that properties and uses it in the multi-binding...

AND you use io.druid.server.initialization.jetty.ChatHandlerServerModule in an extension (like if you have Cli_SOMETHING_CRAZY.java which uses that module), then the properties will not get injected appropriately and therefore will not bind the multi-bindings as you want.

### PR DESCRIPTION
**Pull request by pjain1**
Fixes #2687

### CODE REVIEW

File: server/src/main/java/io/druid/server/initialization/jetty/ChatHandlerServerModule.java [hunk 1]
```diff
@@ -48,9 +47,12 @@
   private static final String MAX_CHAT_REQUESTS_PROPERTY = "druid.indexer.server.maxChatRequests";
   private static final String CHAT_PORT_PROPERTY = "druid.indexer.task.chathandler.port";
 
-  @Inject
   private Properties properties;
```
Comment (guobingkun) [inline server/src/main/java/io/druid/server/initialization/jetty/ChatHandlerServerModule.java]: suggest final

**Comment by guobingkun** [comment]
:+1:

File: server/src/main/java/io/druid/server/initialization/jetty/ChatHandlerServerModule.java (hunk 1 above)
Comment (drcrallen) [inline server/src/main/java/io/druid/server/initialization/jetty/ChatHandlerServerModule.java]: agreed

**Comment by drcrallen** [comment]
:+1: after https://github.com/druid-io/druid/pull/2688#discussion_r56729452 and travis

File: server/src/main/java/io/druid/server/initialization/jetty/ChatHandlerServerModule.java (hunk 1 above)
Comment (pjain1) [inline server/src/main/java/io/druid/server/initialization/jetty/ChatHandlerServerModule.java]: done

**Pull request merged**